scripts/utils.js

```diff
@@ -3,76 +3,96 @@
-      .replace(/đ/g, 'd')
-      .replace(/ħ/g, 'h')
-      .replace(/ı/g, 'i')
-      .replace(/ĸ/g, 'k')
-      .replace(/ŀ/g, 'l')
-      .replace(/ł/g, 'l')
-      .replace(/ß/g, 'ss')
-      .replace(/ŧ/g, 't')
-      .normalize('NFD')
-      .replace(/[^a-z0-9]/g, ''),
+/**
+ * Converts a brand title (as it is seen in simple-icons.json) into a brand
+ * title in HTML/SVG friendly format.
+ * @param {String} brandTitle The title to convert
+ */
+export const titleToHtmlFriendly = (brandTitle) =>
+  brandTitle
+    .replace(/&/g, '&amp;')
+    .replace(/"/g, '&quot;')
+    .replace(/</g, '&lt;')
+    .replace(/>/g, '&gt;')
+    .replace(/./g, (char) => {
+      const charCode = char.charCodeAt(0);
+      return charCode > 127 ? `&#${charCode};` : char;
+    });
 
-  /**
-   * Converts a brand title in HTML/SVG friendly format into a brand title (as
-   * it is seen in simple-icons.json)
-   * @param {String} htmlFriendlyTitle The title to convert
-   */
-  htmlFriendlyToTitle: (htmlFriendlyTitle) =>
-    htmlFriendlyTitle
-      .replace(/&#([0-9]+);/g, (_, num) => String.fromCharCode(parseInt(num)))
-      .replace(
-        /&(quot|amp|lt|gt);/g,
-        (_, ref) => ({ quot: '"', amp: '&', lt: '<', gt: '>' }[ref]),
-      ),
+/**
+ * Converts a brand title in HTML/SVG friendly format into a brand title (as
+ * it is seen in simple-icons.json)
+ * @param {String} htmlFriendlyTitle The title to convert
+ */
+export const htmlFriendlyToTitle = (htmlFriendlyTitle) =>
+  htmlFriendlyTitle
+    .replace(/&#([0-9]+);/g, (_, num) => String.fromCharCode(parseInt(num)))
+    .replace(
+      /&(quot|amp|lt|gt);/g,
+      (_, ref) => ({ quot: '"', amp: '&', lt: '<', gt: '>' }[ref]),
+    );
 
-  /**
-   * Converts a slug into a variable name that can be exported.
-   * @param {String} slug The slug to convert
-   */
-  slugToVariableName: (slug) => {
-    const slugFirstLetter = slug[0].toUpperCase();
-    const slugRest = slug.slice(1);
-    return `si${slugFirstLetter}${slugRest}`;
-  },
+/**
+ * Get contents of _data/simple-icons.json.
+ */
+export const getIconDataString = () => {
+  const __dirname = path.dirname(fileURLToPath(import.meta.url));
+  const rootDir = path.resolve(__dirname, '..');
+  const iconDataPath = path.resolve(rootDir, '_data', 'simple-icons.json');
+  return fs.readFile(iconDataPath, 'utf8');
+};
 
-  /**
-   * Converts a brand title (as it is seen in simple-icons.json) into a brand
-   * title in HTML/SVG friendly format.
-   * @param {String} brandTitle The title to convert
-   */
-  titleToHtmlFriendly: (brandTitle) =>
-    brandTitle
-      .replace(/&/g, '&amp;')
-      .replace(/"/g, '&quot;')
-      .replace(/</g, '&lt;')
-      .replace(/>/g, '&gt;')
-      .replace(/./g, (char) => {
-        const charCode = char.charCodeAt(0);
-        return charCode > 127 ? `&#${charCode};` : char;
-      }),
+/**
+ * Get icon data as object from _data/simple-icons.json.
+ */
+export const getIconData = async () => {
```

Comment:
The same here, don't forget to change all references to them.

```suggestion
export const getIconsData = async () => {
```